MSAL/src/requests/MSALSilentRequest.m

```diff
@@ -227,4 +362,20 @@ - (BOOL)isServerUnavailable:(NSError *)error
     return error.code == MSALErrorUnhandledResponse && responseCode >= 500 && responseCode <= 599;
 }
 
+- (void)updateAppMetadata:(MSIDAppMetadataCacheItem *)appMetadata
```

Comment:
we should also update app metadata when we're using MRRT if we get foci=1 in response

Reply:
App metadata will be updated every time we get a response from the server.